src/main/java/nextstep/courses/domain/session/Session.java

```diff
@@ -0,0 +1,49 @@
+package nextstep.courses.domain.session;
+
+import nextstep.courses.NotEnrollmentPeriodException;
+import nextstep.courses.domain.coverimage.CoverImage;
+import nextstep.payments.domain.Payment;
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Session {
+    private static final String ERR_NOT_PERIOD = "해당 강의는 수강신청 기간이 아닙니다.";
+
+    final Long id;
+    CoverImage coverImage;
+    final List<Long> studentsId = new ArrayList<>();
+    final LocalDate startDate;
+
+    LocalDate endDate;
```

Comment:
```suggestion
    final LocalDate startDate;
    final LocalDate endDate;
```
두개의 상태를 하나의 객체로 도출해보면 어떨까요~?

Reply:
요 부분은 생각지 못했었는데 Schedule class로 분리해보았습니다!